demo/.eslintrc

```diff
@@ -12,6 +12,8 @@
     "class-methods-use-this": 0,
     "no-plusplus": 0,
     "semi": ["error", "never"],
-    "react/require-default-props": "off"
+    "react/require-default-props": "off",
+    "react/jsx-max-props-per-line" : ['error', {maximum: 1}]
```

Comment:
Yes this is why I've only changed this rule as it was raised, have checked all the other components so don't worry